config/environments/deployed_development.rb

```diff
@@ -6,11 +6,9 @@
   # Settings specified here will take precedence over those in config/application.rb.
 
   # Used to handle HTTP_X_WITH_SERVER_DATE header for server side datetime overwrite
-  # TODO: Enable after more thorough testing
-  # config.middleware.use TimeTraveler
+  config.middleware.use TimeTraveler
```

Comment:
We sure this is working too?

Reply:
I will test that on review app too, although now that it it enabled in tests I am a lot more confident it will not break anything.

Reply:
I was definitely seeing a nil error or something somewhere re: this. Whether it came from this middleware I don't know.

Reply:
I wonder if this was just showing up in the stacktrace as a caller?